test/test_pymongo_auth_aws.py

```diff
@@ -81,140 +113,87 @@ def ensure_equal(self, creds, expected):
             self.assertEqual(creds.token, expected['SessionToken'])
         else:
             self.assertEqual(creds.token, expected['Token'])
-        self.assertEqual(creds.expiration, expected['Expiration'])
 
     def test_aws_temp_credentials_env_variables(self):
         os.environ['AWS_ACCESS_KEY_ID'] = 'foo'
         os.environ['AWS_SECRET_ACCESS_KEY'] = 'bar'
-        creds = _aws_temp_credentials()
+        creds = aws_temp_credentials()
         del os.environ['AWS_ACCESS_KEY_ID']
         del os.environ['AWS_SECRET_ACCESS_KEY']
         self.assertEqual(creds.username, 'foo')
         self.assertEqual(creds.password, 'bar')
         self.assertEqual(creds.token, None)
-        self.assertEqual(creds.expiration, None)
-
-    def test_aws_temp_credentials_relative_url(self):
```

Comment:
Is AWS_CONTAINER_CREDENTIALS_RELATIVE_URI no longer supported?

Reply:
It is, internally by `boto3`, but this allowed us to use `localhost` instead of the AWS endpoint.